sites/public/src/md_content/accessibility.md

```diff
@@ -0,0 +1,245 @@
+<style>
+    .bull-list ul{
+        list-style-type: disc;
+    }
+</style>
+<RenderIf language="en">
+
+This is an accessibility statement from the City of Detroit Housing and Revitalization Department. 
+
+### Conformance Status
+
+The [Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/WAI/standards-guidelines/wcag/) defines best practices for designers and developers to improve accessibility of websites for people with disabilities. It defines three levels of conformance: Level A, Level AA, and Level AAA. Our goal is to deliver a web experience that achieves “Level AA” conformance according to the Web Content Accessibility Guidelines v2.1 (WCAG 2.1). We hope to continually iterate and improve Detroit Home Connect even beyond full WCAG conformance.
+
+### Feedback
+
+We welcome your feedback on the accessibility of Detroit Home Connect. Please let us know if you encounter accessibility barriers while using this website by contacting us at the email below:
+
+- E-mail: <detroithomeconnect@detroitmi.gov>
+</div>
```

Comment:
This div shouldn't be there
![image](https://user-images.githubusercontent.com/42942267/228034025-69778164-8dbc-41c1-b9b7-4bb682aad2eb.png)